scripts/start_container.sh

```diff
@@ -3,10 +3,16 @@
 PORT=${PORT:-8888}
```

Comment:
no, though there is a tiny bug in this script:  the line
`docker run -d -p $PORT:8888`
connects the docker port `$PORT` (which should always be open) to localhost:8888 (which might not be open).  Those two should be switched (to `-p 8888:$PORT`), but we have to say which external port to bind to.